tensorflow/python/keras/activations.py

```diff
@@ -302,6 +302,27 @@ def relu(x, alpha=0., max_value=None, threshold=0):
   return K.relu(x, alpha=alpha, max_value=max_value, threshold=threshold)
 
 
+@keras_export('keras.activations.gelu')
```

Comment:
In general, we prefer to only export to v2, as TFv1 is no longer released. Can you update this + the export below to have an empty list for v1? Eg:

```
@keras_export('keras.activations.gelu', v1=[])
```

Reply:
Hi @karmel, does this also apply to `tf.nn.gelu`, like
```python
@tf_export("nn.gelu", v1=[])
```